metadata-io/src/main/java/com/linkedin/metadata/timeseries/elastic/indexbuilder/TimeseriesAspectIndexBuilders.java

```diff
@@ -33,6 +36,12 @@ public void reindexAll() {
     }
   }
 
+  @Override
```

Comment:
Should exist, perhaps not an override but depending on how the clean-up of the ElasticsearchIndex interface goes, you may end up wanting to create a new interface like ElasticSearchIndexedTimeseries (yikes that's long). The timeseries interface applies to timeseries Services' indices (or a subset). Most Services will not be implementing the tiimeseries methods: entity, system metadata, and graph for example do not require the timeseries truncations.